app/src/main/java/com/onfido/android/app/sample/FinalActivity.java

```diff
@@ -57,6 +57,11 @@ private void setFragment(final Fragment fragment) {
     }
 
     @Override
+    public void onPreviousClicked() {
```

Comment:
This function is now required (it wasn't in 1.0.0).
1. Is it also a thing we should include in migration?
2. Should we fill the body of this function in any way (i.e. like in onNextClicked())?
@sergiomarquesmoura @rfreitas

Reply:
No need I think. you can just write `//no op` if you want.

Reply:
It's only required because we are implementing an interface from the SDK on this app (not meant for client developers). But yeah just fill it with // no op and I think we're fine

Reply:
And in a future PR we should clean these things, removing these non-necessary dependencies between the sample app and the SDK @rfreitas

Reply:
CX-1501 created ;)